pom.xml

```diff
@@ -26,38 +26,40 @@
     <distro.openmrsmodulesDir>${distro.baseDir}/openmrs_modules</distro.openmrsmodulesDir>
 
     <!-- Core -->
-    <openMRSVersion>2.1.1</openMRSVersion>
+    <openMRSVersion>2.1.3</openMRSVersion>
 
     <!-- Configs -->
     <openmrsconfigVersion>1.2.0</openmrsconfigVersion>
     <bahmniconfigVersion>1.1.0</bahmniconfigVersion>
 
     <!-- Bahmni Apps -->
-    <bahmniappsRevision>6af59f1</bahmniappsRevision>
+    <bahmniappsRevision>90b64f7</bahmniappsRevision>
 
     <!-- In dev -->
-    <initializerVersion>2.0.0</initializerVersion>
+    <initializerVersion>2.1.0</initializerVersion>
     <bahmnicoreVersion>0.93-SNAPSHOT</bahmnicoreVersion>
+    <postbahmniVersion>1.0.0-SNAPSHOT</postbahmniVersion>
```

Comment:
I've deployed a GHC specific version for Post Bahmni: `1.2.0-GHC-SNAPSHOT`
Could you use it instead?